cmake/CMakeLists.txt

```diff
@@ -316,6 +316,26 @@ if (onnxruntime_USE_TVM)
   if (onnxruntime_USE_LLVM)
     set(USE_LLVM ON)
     add_definitions(-DUSE_TVM_WITH_LLVM)
+
+    # patch for LLVM FCmp assert.
+    # it can be removed once a fix is available.
+    set(TVM_PATCH_COMMAND1 git apply ${CMAKE_SOURCE_DIR}/patches/tvm/FCmp.patch)
```

Comment:
Can we  avoid changing the source directory?

Reply:
Let's create a public forked repo for tvm, like Amazon did.
That will make sync private and public forked tvm repo easier.

Reply:
Wenlei is in the process of sending PR to TVM. I still prefer not to fork.

---
In reply to: [277508559](https://github.com/Microsoft/onnxruntime/pull/881#discussion_r277508559) [](ancestors = 277508559)

Reply:
He just worked for this one. How about others?

Reply:
We still have to fork since they won't take all of the patches.